lib/roo_on_rails/checks/documentation/playbook.rb

```diff
@@ -0,0 +1,23 @@
+require 'roo_on_rails/checks/base'
+
+module RooOnRails
+  module Checks
+    module Documentation
+      class Playbook < Base
+        LOCATION = 'PLAYBOOK.md'.freeze
+
+        def intro
+          'Looking for an on-call Playbook...'
+        end
+
+        def call
+          if File.exist?(LOCATION)
```

Comment:
I copied @danielcooper's structure of:

```
# <service name> Playbook

<intro text>

## Useful Links

<links to Heroku, New Relic, Papertrail, etc.>

## Known Issues

### <alert title>

<how to handle the alert>
```

Seemed like quite a good start anyway.